index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="description"
           content="Application is supposed to help the user manage their free time in a not usuall manner.">
     <meta name="author" content="Red Coders Team">
-    <title>Yawn Killer</title>
+    <title>Yawn Killer best site on web</title>
```

Comment:
Do not exaggerate...it is your subjective opinion...Application's users do not have to share it...;) Perhaps you can add: 
' - **probably** the best site on web ; ]'